python/tvm/target/target.py

```diff
@@ -220,19 +220,24 @@ def intel_graphics(model="unknown", options=None):
     return Target(" ".join(["opencl"] + opts))
 
 
-def micro(hardware="unknown", options=None):
+def micro(model="unknown", options=None):
```

Comment:
Changes for target API ? Should this really be a separate PR ?

Reply:
here i'm really just changing the name of an argument. most people pass this arg as a positional arg, so I don't know that there is much user-visible change. if you really want me to move it outside this change, I can, but jenkins takes 4-6 hours these days and we'd have to write a test here to get it to fail with this change.